doc-changelog/action.yml

```diff
@@ -0,0 +1,138 @@
+name: >
+  Documentation Changelog
+
+description: |
+  Adds a newsfragment to the target repository using ``towncrier``
+  containing the title and number of the pull request.
+
+
+inputs:
+
+  # Required inputs
+
+  path-to-fragment:
+    description: >
+      Path to where the towncrier fragment is stored
+    required: true
+    type: string
```

Comment:
I think this is not required. Projects using this action should configure `towncrier` in their `pyproject.toml`. This is where the fragments directory gets specified. This ensures that running any `towncrier <commnad>` renders a fragment in the desired dir.